lib/tool_belt/release_environment.rb

```diff
@@ -20,20 +20,53 @@ def setup(args = {})
       Dir.chdir(release_directory) do
         @repos.each do |name, repo|
           @systools.execute("git clone #{repo[:repo]} #{name}") if !File.exist?(name.to_s)
-          if github_username
-            Dir.chdir(name.to_s) do
+
+          Dir.chdir(name.to_s) do
+            @systools.execute("git remote set-url origin #{add_username(repo[:repo], github_username)}") if github_username
+
+            if repo[:version_branch]
+              if branch_exists?("origin/#{repo[:branch]}") &&
+                @systools.execute("git checkout #{repo[:branch]}")
```

Comment:
This would checkout the local copy of the branch, if your intent is to checkout the true origin version, maybe explcitily do so? e.g. `git checkout origin/#repo[:branch]`